storage/grpc_client.go

```diff
@@ -1381,16 +1371,43 @@ func (c *grpcStorageClient) NewMultiRangeDownloader(ctx context.Context, params
 			Metageneration:  obj.GetMetageneration(),
 			Generation:      obj.GetGeneration(),
 		},
-		reader: rr,
+		reader: mrd,
 	}, nil
 }
 
-func getActiveRange(r *gRPCBidiReader) []rangeSpec {
-	r.mu.Lock()
-	defer r.mu.Unlock()
-	var activeRange []rangeSpec
-	for k, v := range r.mp {
-		activeRange = append(activeRange, rangeSpec{
+type gRPCBidiReader struct {
```

Comment:
IMO this could also be renamed to something related to MRD given that it's not used by the actual bidi storage.Reader implementation.